scrapy/core/downloader/handlers/http11.py

```diff
@@ -412,9 +412,10 @@ def dataReceived(self, bodyBytes):
 
         if self._maxsize and self._bytes_received > self._maxsize:
             logger.error("Received (%(bytes)s) bytes larger than download "
-                         "max size (%(maxsize)s).",
+                         "max size (%(maxsize)s) in request %(request)s.",
                          {'bytes': self._bytes_received,
-                          'maxsize': self._maxsize})
+                          'maxsize': self._maxsize,
+                          'request': self._request})
```

Comment:
I think this is right and could be also included in this PR.
It is exactly the same way it was done in the next if ([line 425](https://github.com/scrapy/scrapy/blob/master/scrapy/core/downloader/handlers/http11.py#L425)).